tap_ms_graph/streams/group_member.py

```diff
@@ -7,8 +7,9 @@
 
 class GroupMember(FullTableStream):
     tap_stream_id = "group_member"
-    key_properties = ["group_id", "id"]
+    key_properties = ["id"]
```

Comment:
To map child record with parent record we must insert parent id in key_properties.

```suggestion
    key_properties = ["id", "group_id"]
```